autotests/testcrosssigning.cpp

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2023 Tobias Fella <tobias.fella@kde.org>
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+#include <QTest>
+#define private public
+#include "connection_p.h"
+#undef private
```

Comment:
Do you really need this hack? I don't see any private members in `connection_p.h`.